weave

```diff
@@ -1553,25 +1550,10 @@ EOF
     launch)
         deprecation_warnings "$@"
         check_not_running router $CONTAINER_NAME        $BASE_IMAGE
-        check_not_running proxy  $PROXY_CONTAINER_NAME  $BASE_EXEC_IMAGE
         check_not_running_plugin_v2
         COMMON_ARGS=$(common_launch_args "$@")
-        launch_router --plugin "$@"
-        launch_proxy  $COMMON_ARGS
-        ;;
-    launch-router)
-        deprecation_warnings "$@"
-        check_not_running router $CONTAINER_NAME $BASE_IMAGE
-        check_not_running_plugin_v2
-        launch_router "$@"
-        echo $ROUTER_CONTAINER
-        ;;
-    launch-proxy)
-        deprecation_warnings "$@"
-        check_not_running proxy $PROXY_CONTAINER_NAME $BASE_EXEC_IMAGE
-        check_not_running_plugin_v2
-        launch_proxy "$@"
-        echo $PROXY_CONTAINER
+        launch --plugin "$COMMON_ARGS" "$@"
```

Comment:
Done in 974b867dd672e024ad9527dbcaea4c82d3608736 `Add --proxy option to enable/disable proxy.`